helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -592,8 +595,8 @@ private void subscribeForChangesAsyn(NotificationContext.Type callbackType, Stri
 
   private void subscribeForChanges(NotificationContext.Type callbackType, String path,
       boolean watchChild) {
-    logger.info("Subscribing changes listener to path: {}, type: {}, listener: {}", path,
-        callbackType, _listener);
+    logger.info("Subscribing changes listener to path: {}, callback type: {}, event types: {}, "
```

Comment:
For debug, please add new fields to the end of the string. And don't change the existing content if they are not hurt too much. Otherwise, the dev needs to switch the code version back and forth to double-check before debugging the older version deployment.